cogs/stats.py

```diff
@@ -359,9 +354,12 @@ async def server_stats(self, ctx: TeXBotApplicationContext) -> None:
                         message_counts["roles"][f"@{author_role_name}"] += 1
 
         too_few_roles_stats: bool = math.ceil(max(message_counts["roles"].values()) / 15) < 1
-        too_few_channels_stats: bool = math.ceil(
-            max(message_counts["channels"].values()) / 15,
-        ) < 1
+        too_few_channels_stats: bool = (
+            math.ceil(
+                max(message_counts["channels"].values()) / 15,
+            )
+            < 1
+        )
```

Comment:
* Boolean flag required capitalisation
* Boolean flag requires annotating as `Final`
* Splitting mid-expression indexing harms readability. I agree that the comparison statement should be moved onto the same line, however.

Fix:
```python
TOO_FEW_CHANNEL_STATS: Final[bool] = (
    math.ceil(max(message_counts["channels"].values()) / 15) < 1  # noqa: E501
)